docs/dv/09_charon_cli_reference.md

```diff
@@ -16,8 +16,13 @@ The following is a reference for charon version [`v0.10.0`](https://github.com/O
 
 The following are the top-level commands available to use. 
 
-```markdown
-charon help
+### Help with charon commands
+
+The `charon help` prints the commands available in charon.
+
+```shell
+foo@bar:~$ charon help
```

Comment:
Not really sure `shell` is better here, it only highlights weird phrases that mean things in sh script lang 
![Uploading Screenshot_20220825_111959_org.mozilla.firefox.jpg…]()